state.go

```diff
@@ -828,6 +839,9 @@ func NewErrorState(err menderError) State {
 }
 
 func (e *ErrorState) Handle(ctx *StateContext, c Controller) (State, bool) {
+	// stop deployment logging
+	DeploymentLogger.Disable()
```

Comment:
Should we disable the logger here? Won't that hide error messages from the deployments? Or are all those handled in `ReportErrorState`?

Reply:
So there are two different error states: `ErrorState` and `UpdateErrorState`. For the later we are logging data.

Reply:
Ah I see.